composer.json

```
@@ -4,7 +4,7 @@
     "require": {
         "php": "^7.4|^8.1",
```

```suggestion
        "php": "^7.4|^8.1|^8.2",
```
you said they now support 8.2